Assets/Scripts/Button Cell.cs

```diff
@@ -0,0 +1,50 @@
+
+using UnityEngine;
+using TMPro;
+using UnityEngine.UI;
+
+public class ButtonCell : Cell
+{
+    [SerializeField] private TextMeshProUGUI cellText;
+    [SerializeField] private Image image;
+    private System.Action callback;
+    private ButtonCellData bData = new();
+
+    public override void Configure(CellData data)
```

Comment:
Make generic structure

Reply:
After a discussion it was decided not to go to down this path

Reply:
A ticket to address this was created in the backlog